notebooks/calling_peekingduck_in_python.ipynb

```diff
@@ -184,7 +184,7 @@
     "This grants us the flexibility of inspecting the data for debugging or interacting with the data to integrate the inference loop external processes.\n",
     "\n",
     "We have also defined a utility function for retrieving the image region of the license plate with a highest confidence score to improve code clarity.\n",
-    "For more information on how to convert between bounding box and image coordinates, please refer to the [\"Bounding Box vs Image Coordinates\"](https://peekingduck.readthedocs.io/en/docs-v1.2/tutorials/01_hello_cv.html#bounding-box-vs-image-coordinates) section in our tutorials."
+    "For more information on how to convert between bounding box and image coordinates, please refer to the [\"Bounding Box vs Image Coordinates\"](https://peekingduck.readthedocs.io/en/stable/tutorials/01_hello_cv.html#bounding-box-vs-image-coordinates) section in our tutorials."
    ]
   },
   {
```

Comment:
The title of this notebook is still `Import PeekingDuck as a Module Demo`, best to update as well for consistency.

Reply:
Updated, changed the filename/title in the Colab notebook as well